docs/root/intro/arch_overview/upstream/aggregate_cluster.rst

```diff
@@ -0,0 +1,103 @@
+Aggregate cluster uses tiered load balancing. The load balancer chooses cluster and piority first and then delegates the load balancing to the load balancer of the selected cluster. 
+The top level load balancer reuses the existing load balancing algorithm by linearizing the priority set of multiple clusters into one. 
+
+Linearize Priority Set
+----------------------
+
+Upstream hosts are divided into multiple :ref:`priority levels <arch_overview_load_balancing_priority_levels>` and each priority level contains 
+a list of healthy, degraded and unhealthy hosts. Linearization is used to simplify the host selection during load balancing by merging priority levels 
+from multiple clusters. For example, primary cluster has 3 priority levels, secondary has 2 and tertiary has 2 and the failover ordering is 
+primary, secondary, tertiary. 
+
++-----------+----------------+-------------------------------------+
+| Cluster   | Priority Level |  Priority Level after Linearization |
++===========+================+=====================================+
+| Primary   | 0              |  0                                  |
++-----------+----------------+-------------------------------------+
+| Primary   | 1              |  1                                  |
++-----------+----------------+-------------------------------------+
+| Primary   | 2              |  2                                  |
++-----------+----------------+-------------------------------------+
+| Secondary | 0              |  3                                  |
++-----------+----------------+-------------------------------------+
+| Secondary | 1              |  4                                  |
++-----------+----------------+-------------------------------------+
+| Tertiary  | 0              |  5                                  |
++-----------+----------------+-------------------------------------+
+| Tertiary  | 1              |  6                                  |
++-----------+----------------+-------------------------------------+
+
+Example
+-------
+
+A sample aggregate cluster configuration could be:
+
+.. code-block:: yaml
+
+  name: aggregate_cluster
+  connect_timeout: 0.25s
+  lb_policy: CLUSTER_PROVIDED
+  cluster_type:
+    name: envoy.clusters.aggregate
+    typed_config:
+      "@type": type.googleapis.com/envoy.config.cluster.aggregate.v2alpha.ClusterConfig
+      clusters:
+      # cluster primary, secondary and tertiary should be defined outside.
+      - primary
+      - secondary
+      - tertiary
+
+Note: :ref:`PriorityLoad retry plugins <envoy_api_field_route.RetryPolicy.retry_priority>` won't work for aggregate cluster because the aggregate load balancer
+will override the *PriorityLoad* during load balancing.
+
+
+Load Balancing Example
+----------------------
+
+Aggregate cluster uses tiered load balancing algorithm and the top tier is distributing traffic to different clusters according to the health score across 
+all priorities in each cluster.
```

Comment:
Can you also link to https://299913-65214191-gh.circle-artifacts.com/0/source/generated/docs/intro/arch_overview/upstream/load_balancing/priority.html here? I know you do that above as well, but it is useful to repeat.

Reply:
done